src/common/base-store.ts

```diff
@@ -48,16 +46,27 @@ export abstract class BaseStore<T = any> extends Singleton {
     return when(() => this.isLoaded);
   }
 
-  protected constructor(protected params: BaseStoreParams) {
+  protected constructor(protected params: BaseStoreParams<T>) {
     super();
     makeObservable(this);
+  }
+
+  /**
+   * This must be called after the last child's constructor is finished (or just before it finishes)
+   */
+  load() {
```

Comment:
Given that this is now synchronous I have made it so that all the in-repo stores initialize in the constructor.

This several benefits:
1. It is inline with how the `conf` package works so we have less dissonance.
2. We can get rid of `BaseStore.isLoaded`
3. No need to `await` the loading so config loading is now always done in a static order.

It is fine to use synchronous code here because loading only happens once.